src/content/configuration/optimization.md

```diff
@@ -324,4 +324,37 @@ module.exports = {
     concatenateModules: true
   }
 };
-```
\ No newline at end of file
+```
+
+## `optimization.sideEffects`
+
+`bool`
+
+Tells webpack to recognise the [sideEffects](https://github.com/webpack/webpack/blob/75c6416cfe95217b5ce60d02ba5d24d88c50f538/examples/side-effects/README.md) flag in `package.json` or rules to skip over modules which are flagged to contain no side effects when exports are not used. 
```

Comment:
not sure whether i want to link master or target current version via commit reference :S